### PR TITLE
Fix/nex 944/auth on test resume

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/NEX-944

### CODE REVIEW

File: model/authorization/RemoteProcotoredSessionContext.php [hunk 1]
```diff
@@ -44,4 +48,20 @@ public function getDeliveryExecutionId(): string
...
+
+    /**
+     * @throws SecurityException
+     */
+    public function consume(): void
+    {
+        if ($this->consumed) {
+            throw new SecurityException('Proctorio authorisation attempt has been already consumed');
+        }
+        $this->consumed = true;
```
Comment (gabrielfs7) [inline model/authorization/RemoteProcotoredSessionContext.php:60]:
```suggestion
        $this->consumed = true;
```

This exception will never be thrown.

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**